tsconfig.json

```diff
@@ -6,6 +6,8 @@
     "declaration": true,
     "importHelpers": true,
     "resolveJsonModule": true,
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
Just curious, is this because of a rough edge in @subql or something?

Reply:
It's actually Agoric deps. I've filed an issue,
- https://github.com/Agoric/agoric-sdk/issues/11038